aad/spring-security/reactive/webflux/oauth2/spring-cloud-gateway/resource-server-2/src/main/resources/application.yml

```diff
@@ -0,0 +1,19 @@
+# Please fill these placeholders before run this application:
+# 1. ${tenant-id}
+# 2. ${resource-server-2-client-id}
+
+logging:
+  level:
+    root: DEBUG
+server:
+  port: 8082
+spring:
+  security:
+    oauth2:
+      resourceserver:
+        jwt:
+          jwk-set-uri: https://login.microsoftonline.com/${tenant-id}/discovery/v2.0/keys
+          issuer-uri: https://login.microsoftonline.com/${tenant-id}/v2.0
+          audiences: ${resource-server-2-client-id}
```

Comment:
I suggest using the upper case here for variables in yaml, 
${resource-server-2-client-id} --> ${RESOURCE-SERVER-2-CLIENT-ID}

1. our spring cloud [azure](https://microsoft.github.io/spring-cloud-azure/docs/current/reference/html/index.html#auto-configure-azure-sdk-clients) doc follow this pattern.
2.other samples I refactored using the upper case.
it's better to be consistent.

Reply:
Updated.